src/coreclr/jit/lclmorph.cpp

```diff
@@ -382,41 +327,26 @@ class LocalAddressVisitor final : public GenTreeVisitor<LocalAddressVisitor>
                     return false;
                 }
 
-                m_lclNum  = val.m_lclNum;
-                m_offset  = newOffset.Value();
-                m_address = true;
+                m_lclNum = val.m_lclNum;
+                m_offset = newOffset.Value();
             }
 
-            INDEBUG(val.Consume();)
+            INDEBUG(val.Consume());
             return true;
         }
 
-        //------------------------------------------------------------------------
-        // Indir: Produce a location value from an address value.
-        //
-        // Arguments:
-        //    val       - the input value
-        //    addOffset - the offset to add
-        //
-        // Return Value:
-        //    `true` if the value was consumed. `false` if the input value
-        //    cannot be consumed because it is itsef a location or because
-        //    the offset overflowed. In this case the caller is expected
-        //    to escape the input value.
-        //
-        // Notes:
-        //   - LOCATION(lclNum, offset) => not representable, must escape
-        //   - ADDRESS(lclNum, offset) => LOCATION(lclNum, offset + addOffset)
-        //     if the offset overflows then location is not representable, must escape
-        //   - UNKNOWN => UNKNOWN
-        //
-        bool Indir(Value& val, unsigned addOffset = 0)
+        bool AddOffset(unsigned addOffset)
```

Comment:
A bit of code duplication (and no header comment), but this will be deleted in #85568.